src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
 body {
   margin: 0;
+  font-family: Arial;
+  font-size: 100px;
+}
+
+$size: 300px;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax($size,1fr));
+  color: #fff;
+}
+
+@media only screen and (min-width: 1200px) {
+  .container {
+    grid-template-columns: repeat(auto-fit, minmax($size+1, 1fr));
```

Comment:
You have appropriate blocks location with min-width: 1200px, but with more large monitors your blocks are located on one row. Try to use repeat(3, 1fr) instead of a line, which is written on line with number 17.

Reply:
Yes, it`s better. Thanks